Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: local test build
+
+# used with another locally situated consuming codebase to test changes.
+local:
+	./gradlew clean build publishToMavenLocal
+
+build:
+	./gradlew build
+
+# run tests
+test:
+	./gradlew test
```

Comment:
What is this used for?

Reply:
I'm using it for shortcuts to commonly run commands.